### DISCUSSION TITLE
Are there plans for the 1.21 version?

### DISCUSSION THREAD

**Discussion by IsaelcuevasMendez** [vectorwing/FarmersDelight#933] [Q&A] [answered]
I'm making a server with a friend, and I want to put this mod in, do you think update it?

**Comment by TheGridExpert** [comment, marked as answer]
1.21 is in development right now. It will be ported, but it will take time

**Comment by vectorwing** [comment]
1.21 is out! Go get it! https://modrinth.com/mod/farmers-delight/version/1.21-1.2.4
